app/views/transaction_items/index.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if @transactions.present? %>
+<%@transactions.each do |transaction|%>
+  <div class="col-md-12 mb-3">
+    <div class="card w-100">
+      <div class="card-body d-flex">
+        <div class="flex-grow-1 name">
+            <%= transaction.transaction_item.name %>          
+        </div>
+        <div class="text-right amount">
+            <%= transaction.transaction_item.amount %>
+        </div>
+      </div>
+    </div>
+  </div>
+  <%end%>
+<% end %>
+<div class="btn-container">
+    <%= link_to new_category_transaction_item_path(category_id: params[:category_id]), class:"link-container" do %>
+        <button class="btn btn-success">Add Transaction</button>
+    <% end %>
+</div>
```

Comment:
- [ ] Kindly position this button to the bottom section of the screen so that your design can match the provided Behance design.

![image](https://user-images.githubusercontent.com/86318284/219852363-a3b46bc8-db21-41cd-b5c0-9e7df771b9e9.png)

![image](https://user-images.githubusercontent.com/86318284/219852408-7229247b-f21f-41f8-9788-c3e59d83e02e.png)